### ISSUE TITLE
dhall-docs: Syntax highlighting for examples and source

### ISSUE THREAD

**Issue by darichey** [dhall-lang/dhall-haskell#2359]
I think this would be a nice addition to the docs tool, but I wanted to open this issue and get some feedback before working on it more.

My initial experiments with prismjs yielded fairly promising results:
![image](https://user-images.githubusercontent.com/3052455/147891957-1a2a06c2-3362-47ad-aa6a-98d63fcd1d39.png)

Would this be a welcome change?

**Comment by Gabriel439** [comment]
Yeah, definitely!  We originally had syntax highlighting at one point but if I remember correctly we had to remove it in the course of adding some other feature.  Restoring syntax highlighting would be perfect!

**Comment by kukimik** [comment]
>  if I remember correctly we had to remove it in the course of adding some other feature

It was removed here: #1959.
